packages/client/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this npm module are documented in this file.
 
+## [8.0.1] - 2024-03-2x
+
+- Blah
```

Comment:
Blah blahblah? blah blah!

Reply:
Yeah, that was not really finished. 😊 I added entries in all CHANGELOGs that required adjustment and also fixed some minor errors in the READMEs (example links were wrong).